qa-tests-backend/src/test/groovy/ImageManagementTest.groovy

```diff
@@ -78,14 +78,14 @@ class ImageManagementTest extends BaseSpecification {
     def "Verify two consecutive latest tag image have different scans"() {
         given:
         // Scan an ubuntu 14:04 image we're pretending is latest
-        def img = Services.scanImage(
+        def img = ImageService.scanImage(
             "quay.io/rhacs-eng/qa-multi-arch:ubuntu-latest" +
-                "@sha256:64483f3496c1373bfd55348e88694d1c4d0c9b660dee6bfef5e12f43b9933b30") // 14.04
+                "@sha256:64483f3496c1373bfd55348e88694d1c4d0c9b660dee6bfef5e12f43b9933b30", false) // 14.04
```

Comment:
Why do we need `false` now?

Reply:
Ah, got it now

Reply:
Maybe we should have a dedicated method instead of flag? This `false` tells nothing. How about `scanImageNoSnoozed`?